hm_pyhelper/exceptions.py

```diff
@@ -1,3 +1,5 @@
+
```

Comment:
Is this extra space intentional?

Reply:
Github was giving me a warning saying no new line present? Not sure why but to remove the warning I added it